### PR TITLE
Add rootlesskit-config-nonroot package.

### PR DESCRIPTION
**Pull request by wlynch**
Rootlesskit requires configuration to carve out UID namespaces. This adds a configuration for the user "nonroot" which is a common user we use for images.

See https://rootlesscontaine.rs/getting-started/common/subuid/ for more details.

Related: https://github.com/wolfi-dev/os/pull/6616

### CODE REVIEW

**Comment by vaikas** [comment]
I would think we would want this to be a subpackage of rootlesskit?
https://wiki.inky.wtf/docs/teams/engineering/images/package-and-image-guides/serviceable-packages/

And then folks could override it?
https://wiki.inky.wtf/docs/teams/engineering/images/package-and-image-guides/customizing-configurations/

**Comment by wlynch** [comment]
oooo nice! Done.

(we should consider adding these docs to the wolfi repo)

**Review by imjasonh — DISMISSED** [review]

File: rootlesskit.yaml [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
   name: rootlesskit
   version: 1.1.1
-  epoch: 0
+  epoch: 3
   description: RootlessKit is a Linux-native implementation of "fake root" using user_namespaces(7).
   copyright:
     - license: Apache-2.0
+  dependencies:
+    # Let's ensure that we have config. This gets the default, but can also
+    # be override by users.
```
Comment (vaikas) [inline rootlesskit.yaml:10]: super nit my stupid typo s/override/overridden/

File: rootlesskit.yaml [hunk 2]
```diff
@@ -1,10 +1,15 @@
 package:
   name: rootlesskit
   version: 1.1.1
-  epoch: 0
+  epoch: 3
```
Comment (vaikas) [inline rootlesskit.yaml:4]: nit, maybe change this back to 1? I assume this is because of local devl / changes? 🤣

File: rootlesskit.yaml (hunk 1 above)
Comment (wlynch) [reply rootlesskit.yaml]: Done.

File: rootlesskit.yaml (hunk 2 above)
Comment (wlynch) [reply rootlesskit.yaml]: Done.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**